readthedocs/organizations/querysets.py

```diff
@@ -137,13 +128,11 @@ def clean_artifacts(self):
         are disabled and their artifacts weren't cleaned already. We should be
         safe to cleanup all their artifacts at this point.
         """
-        end_date = timezone.now().date() - timedelta(days=3 * DISABLE_AFTER_DAYS)
-        orgs = self.filter(
+        return self.subscription_ended(days=3 * DISABLE_AFTER_DAYS, exact=True).filter(
```

Comment:
```suggestion
        return self.subscription_ended(days=3 * DISABLE_AFTER_DAYS, exact=False).filter(
```

The docstring says "These organizations are **at least** 3*DISABLE_AFTER_DAYS", so I understand we don't want the exact day here.